src/ViewLayer/DisplayDashboard/DisplayDashboardView/DisplayDashboardView.cpp

```diff
@@ -169,6 +169,27 @@ void DisplayDashboardView::aliveReceived(bool)
 }
 void DisplayDashboardView::prechargeStateReceived(QString prechargeState)
 {
+    if(prechargeState=="CHARGE_ENGAGE")
```

Comment:
Make sure the if statements follow the same logic as the batteryView.cpp statements